modules/userinterfacegl/include/modules/userinterfacegl/glui/widgets/rangeslider.h

```diff
@@ -73,6 +73,11 @@ class IVW_MODULE_USERINTERFACEGL_API RangeSlider : public Element {
     void setMinSeparation(int sep);
     int getMinSeparation() const;
 
+    double getHandleWidth() const;
+
+    bool getShowGroove() const;
```

Comment:
Simply showGroove?

Reply:
that is also a bit ambiguous since it's a verb...